### ISSUE TITLE
export builds

### ISSUE THREAD

**Issue by WarenGonzaga** [WarenGonzaga/css-text-portrait-builder#10]
It is nice to have an output package that can be shared with your friends.

**Comment by WarenGonzaga** [comment]
Still thinking of using GulpJS to achieve this. I don't know if Parcel can package the project that can work without a server.

**Comment by WarenGonzaga** [comment]
I will not include this for now since I have problems with Yarn and Gulp.

**Comment by WarenGonzaga** [comment]
This is the reason why I have problems with Yarn #43

**Comment by decanTyme** [comment]
> Still thinking of using GulpJS to achieve this. I don't know if Parcel can package the project that can work without a server.

I think we can just export and compress Parcel builds since they do contain the necessary `html` file, then it's up to the user how they will serve/showcase their exported builds. They can just open it manually in a browser if they like, tho when I tried it it didn't work until I manually corrected the `src` paths in the `index.html` and removed `type="module"` because:

![image](https://user-images.githubusercontent.com/74456102/143681817-5b0717be-cbf4-4107-af0b-94b1c85ba65c.png)

But if they serve it with a local server it [should be fine](https://stackoverflow.com/a/10752078/17149207). They can also just directly edit these builds since it should contain all necessary deps, albeit minified.

Now, If you want this to be as open as possible to the _non-technical_ we would have to build some sort of script to mitigate the issue above. Or something like [Web Server for Chrome](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb?hl=en). I don't see any other solutions as of yet.

What are your thoughts on this? How would they get their builds? Downloadable links?
